test/transaction.cpp

```diff
@@ -0,0 +1,163 @@
+/*
+    This file is part of cpp-ethereum.
+
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file transaction.cpp
+ * @author Christoph Jentzsch <cj@ethdev.com>
+ * @date 2014
+ * Transaaction test functions.
+ */
+
+#include <boost/filesystem/operations.hpp>
+#include <boost/test/unit_test.hpp>
+#include "JsonSpiritHeaders.h"
+#include <libdevcore/CommonIO.h>
+#include <libethereum/BlockChain.h>
+#include <libethereum/State.h>
+#include <libethereum/ExtVM.h>
+#include <libethereum/Defaults.h>
+#include <libevm/VM.h>
+#include "TestHelper.h"
+
+using namespace std;
+using namespace json_spirit;
+using namespace dev;
+using namespace dev::eth;
+using namespace dev::eth;
+
+namespace dev {  namespace test {
+
+bytes importByteArray(const std::string& str)
```

Comment:
please put this function in TestHelper.cpp and see where I have used the code, so it can be replaced by this function, so we keep DRY (don't repeat yourself).